.env.example

```
@@ -0,0 +1,20 @@
+# --- Server-side secrets (not exposed to client)
+
+# Admin Credentials
+NUXT_ADMIN_USERNAME=admin
+NUXT_ADMIN_PASSWORD=123
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NUXT_ADMIN_PASSWORD key should go before the NUXT_ADMIN_USERNAME key

(UnorderedKey)